stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -3542,59 +3541,139 @@ impl SortitionDB {
         sortition_id: &SortitionId,
         rc_info: &RewardCycleInfo,
     ) -> Result<(), db_error> {
-        let sql = "INSERT INTO preprocessed_reward_sets (sortition_id,reward_set) VALUES (?1,?2)";
+        let sql = "REPLACE INTO preprocessed_reward_sets (sortition_id,reward_set) VALUES (?1,?2)";
         let rc_json = serde_json::to_string(rc_info).map_err(db_error::SerializationError)?;
         let args: &[&dyn ToSql] = &[sortition_id, &rc_json];
         sort_tx.execute(sql, args)?;
         Ok(())
     }
 
+    /// Get the prepare phase start sortition ID of a reward cycle
+    fn inner_get_prepare_phase_start_sortition_id_for_reward_cycle(
+        index_conn: &SortitionDBConn,
```

Comment:
Rather than `Self::inner_*` for these functions, it'd be simpler to just make these instance functions of SortitionDBConn, then there wouldn't be a namespace overlap and they could just be called `get...()`, and they'd be less verbose to invoke.

Reply:
Sure